protocol/clproto_cpp/include/clproto.h

```diff
@@ -27,6 +40,7 @@ enum MessageType {
   JOINT_TORQUES_MESSAGE = 13,
   SHAPE_MESSAGE = 14,
   ELLIPSOID_MESSAGE = 15,
+  PARAMETER_MESSAGE = 16
```

Comment:
No comma at the end?

Reply:
probably not that is the last one of the enum